app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -48,6 +48,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
           }
         }
       })
+
+      $(scope).on('change', '[data-exclusive=true] input[type=checkbox]', function (e) {
+        var currentCheckbox = e.target
+        var checkboxes = currentCheckbox.closest('.govuk-checkboxes')
+        var exclusiveOption = $(checkboxes).find('input[type=checkbox][data-exclusive]')
+        var nonExclusiveOptions = $(checkboxes).find('input[type=checkbox]:not([data-exclusive])')
+
+        if (currentCheckbox.dataset.exclusive === 'true' && currentCheckbox.checked === true) {
+          nonExclusiveOptions.each(function () {
+            $(this).prop('checked', false)
+          })
```

Comment:
Does it work to do `nonExclusiveOptions.prop('checked', false)` here?

Reply:
Yep, it would work well in our case, as there's only one exclusive option, but I thought a loop would make the script resilient in case the exclusive option is added to more than one item.